C/tests/c4CertificateTest.cc

```diff
@@ -35,7 +35,7 @@ TEST_CASE("C4Certificate smoke test", "[Certs][C]") {
     for (int i = 0; c4cert_subjectNameAtIndex(certs.temporaryClientIdentity.cert, i, &name); ++i) {
         C4Log("  %.*s = '%.*s'", SPLAT(name.id), SPLAT(name.value));
         if (i == 0) {
-            CHECK(name.id == kC4Cert_CommonName);
+            CHECK((name.id == kC4Cert_CommonName));
```

Comment:
Just curious, why change to double brackets?

Reply:
I guess I forgot to write about this....the upgraded version of Catch does some funky things to try to break down the expression in these macros, but it has trouble when using `==` to compare `C4Slice` instances.  In order to make it stop freaking out, I wrapped the whole expression in parenthesis so that it would stop trying to break it down.